content/docs/started/cloud/getting-started-icp.md

```diff
@@ -1,265 +1,11 @@
-
-kubectl -n kube-system scale deploy logging-elk-client --replicas=0
-kubectl -n kube-system scale deploy logging-elk-kibana --replicas=0
-kubectl -n kube-system scale deploy logging-elk-logstash --replicas=0
-kubectl -n kube-system scale deploy logging-elk-master --replicas=0
-kubectl -n kube-system scale deploy secret-watcher --replicas=0
-```
-
-### Deployed Service's EXTERNAL-IP field stuck in PENDING state
-If you have deployed a service of `LoadBalancer` type, but the EXTERNAL-IP field stucks in `<pending>` state, this is because IBM Cloud Private doesn't provide a built-in support for the `LoadBalancer` service. To enable the `LoadBalancer` service on IBM Cloud Private, please see options described [here](https://medium.com/ibm-cloud/working-with-loadbalancer-services-on-ibm-cloud-private-26b7f0d22b44).
-
-
-### CustomResourceDefinition.apiextensions.k8s.io "profiles.kubeflow.org" is invalid during v0.7 installation
-If you install Kubeflow v0.7 in IBM Cloud Private 3.1.0, you may encounter the following error:
-```
-failed to apply:  (kubeflow.error): Code 500 with message: kfApp Apply failed for kustomize:  (kubeflow.error): Code 500 with message: Apply.Run  Error error when creating "/tmp/kout904105401": CustomResourceDefinition.apiextensions.k8s.io "profiles.kubeflow.org" is invalid: spec.validation.openAPIV3Schema: Invalid value:...
-must only have "properties", "required" or "description" at the root if the status subresource is enabled
-```
-The default Kubernetes version in IBM Cloud Private 3.1.0 is 1.11 which is incompatilbe with Kubeflow v0.7. Please upgrade IBM Cloud Private with a Kubernetes version that is compatible with the Kubeflow version as described in [Minimum system requirements](https://www.kubeflow.org/docs/started/k8s/overview/).
-
-### v.07 Kubeflow pods stuck in CreateContainerConfigError state
-If you install Kubeflow v0.7 in IBM Cloud Private 3.2.0, you may find pods stuck in CreateContainerConfigError state and describe pod show the following error: 
-``` 
-Error: container has runAsNonRoot and image will run as root
-```
-If you install IBM Cloud Private version 3.2.0 or later as a new installation, the default pod security policy setting is the [ibm-restricted-psp policy](https://www.ibm.com/support/knowledgecenter/en/SSBS6K_3.2.0/manage_cluster/security.html), which is applied to all of the existing and newly created namespaces. You can bypass this by deploying Kubeflow using a non-root user id. You can also use [IBM Cloud private CLI cm commands](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.2.0/manage_cluster/cli_cm_commands.html) to change the default setting.  See [Pod isolation](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.2.0/user_management/iso_pod.html) for other options.
+Please refer to the [install section](/docs/ibm/install-kubeflow) in the
```

Comment:
Change "install" to "installation", because "install" is the verb whereas "installation" is the noun.

Reply:
Thanks @sarahmaddox  for your review!
I have modified the file. Please review/approve. Feel free to let me know if further change is required.

Reply:
Thanks!